src/main/java/faang/school/godbless/FeedPeterGriffin/FoodDeliveryTask.java

```diff
@@ -0,0 +1,36 @@
+
+package faang.school.godbless.FeedPeterGriffin;
+
+import java.util.Random;
+
+public class FoodDeliveryTask implements Runnable {
+    private String character;
+    private int foodAmount;
+
+    public FoodDeliveryTask(String character, int foodAmount) {
+        this.character = character;
+        this.foodAmount = foodAmount;
+    }
+
+    @Override
+    public void run() {
+        Random random = new Random();
+        String foodType = getFoodType();
+        StringBuilder message = new StringBuilder(character + " is receiving " + foodAmount + " " + foodType + "\n");
```

Comment:
You could use String.format(), that's a better approach, since you don't put anything additional into the String Pool.